addons/web/static/src/public/colibri.js

```diff
@@ -259,12 +274,20 @@ export class Colibri {
                 nodes = this.getNodes(sel);
                 this.dynamicNodes.set(sel, nodes);
             }
+            const toReplay = [];
             const descr = content[sel];
             for (const directive in descr) {
                 const value = descr[directive];
                 if (directive.startsWith("t-on-")) {
                     const ev = directive.slice(5);
                     const [event, handler, options] = this.addListener(nodes, ev, value);
+                    if (event === "click" && this.bufferedClicks.length) {
+                        for (const { ev, target } of this.bufferedClicks) {
+                            if (target.matches(sel)) {
```

Comment:
I might be mistaken... but if the selector matches several elements, and distinct interactions are set up, wouldn't these potentially be called on the wrong one ?
Oh right, there is a single handler, but then shouldn't we re-inject the target ? (or was only `currentTarget` vanishing - but then, why do we collect those separately from events initially)
Anyway, if `.withTarget` is used, there might be such an issue.